mindsdb/integrations/handlers/statsforecast_handler/statsforecast_handler.py

```diff
@@ -133,14 +141,20 @@ def predict(self, df, args={}):
         sf = StatsForecast(models=[], freq=model_args["frequency"], df=training_df)
         sf.fitted_ = fitted_models
         forecast_df = sf.predict(model_args["horizon"])
-        forecast_df = forecast_df[forecast_df.index.isin(groups_to_keep)]
-        return get_results_from_nixtla_df(forecast_df, model_args)
+        results_df = forecast_df[forecast_df.index.isin(groups_to_keep)]
```

Comment:
It appears that the original `results_df` is not needed to reconcile, if so, I'd recommend defining the non-reconciled definition inside an `else` clause to adhere to `mindsdb` coding guidelines (I will share them with you, we don't really enforce them but it's a nice to have).

Reply:
Makes sense - have addressed that now :)